.editorconfig

```diff
@@ -0,0 +1,39 @@
+root = true
```

Comment:
this is primarily to prevent from auto formatting of yaml files in github workflows on vscode